standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+
+Note that cross platform app development, where in any way feasible, **is always preferable to native app development, even when we are currently targeting only a single platform (say Apple) and range of associated devices**. That is because such apps ought to be easily adaptable to run on a different platform, for example if our selected choice of devices changes, or if we decide to make the app, or a version of the app, available on more devices. Furthermore, investment, in terms of time, money and "hard lessons learned", in cross platform app development is more likely to be applicable across the whole range of possible future apps than if we focus on a single native app technology.
+
+### Option 3: Build a Native App
+
+As a final fallback option it is therefore permissible to develop a native application. **Should the app be intended for anything other than purely and permanently internal use, this will involve developing and maintaining two entirely separate versions of the app, using the technologies native to the given platform: meaning using the Swift language and the Xcode IDE for Apple, and either the Java or Kotlin languages for Android** (Android Studio is the free and recommended IDE for such development but, as this is a much more open eco-system, it cannot be mandated).
+
+**A native app developed purely and permanently for internal use should be developed in Swift and Xcode alone.**
+
+Note that Swift is essentially a replacement for an earlier Apple language called Objective-C. Objective-C is a notoriously eccentric programming language and, as it is now also arguably approaching obsolescence, **its use must be avoided on any new projects.**
+
+Troubleshooting
+---------------
+
+Often, issues will only be encountered when running apps on actual target devices. **Therefore you must have the knowledge, kit, privileges and app distribution arrangements in place to be able to remotely debug the app on suitable devices.**
```

Comment:
Yes. 💯 

Put this at the top of the list!

Reply:
If you insist!

Reply:
Now placed just after standards on "what is a mobile app" and supported platforms, as those are scene setters really.